### PR TITLE
[Docs] fix: Compatibility Matrix test result for Components and typo's

### PR DESCRIPTION
**Pull request by singh1203**
**Notes for Reviewers**

This PR fixes #https://github.com/meshery/meshery/issues/10335

Before:
Screenshot from 2024-02-16 12-36-56

After:
Screenshot from 2024-02-18 00-18-37

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by singh1203** [comment]
Note for Reviewer: Also fixed some typos and link.

**Comment by l5io** [comment]
🚀 Preview for commit fb4c8de042aa337de9b30332ac10b5735ff3f1e5 at: https://65d101ea026f86cfc293b1f9--meshery-docs-preview.netlify.app

File: docs/_layouts/past-results-layout.html [hunk 1]
```diff
@@ -92,5 +92,5 @@ <h2>{{page.subheading}} Integration Tests</h2>
   </div>
 </div>
 
-{% include_cached compatibility/past-results.html %}
+{% include compatibility/past-results.html %}
```
Comment (leecalcote) [inline docs/_layouts/past-results-layout.html:95]: Pass variables as parameters, please.

Let's continue to use the Build Cache plugin. See that plugin for instructions on how to do this.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit de6fcf5742ad081377b99d4652e2868b5a54b43b at: https://65d230ae0e24596dd204dc4c--meshery-docs-preview.netlify.app

File: docs/_layouts/past-results-layout.html (hunk 1 above)
Comment (singh1203) [reply docs/_layouts/past-results-layout.html]: Done @leecalcote.

**Review by iArchitSharma — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 3cb120d033c9887f369e8ef335771f233d355972 at: https://65d31cd4a5c298f8957c1213--meshery-docs-preview.netlify.app

**Comment by singh1203** [comment]
@iArchitSharma Thank you so much for correcting.

**Comment by vishalvivekm** [comment]
@singh1203 let's discuss this in websites call today. I'll be adding this in agenda items. https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit

**Comment by singh1203** [comment]
> @singh1203 let's discuss this in websites call today. I'll be adding this in agenda items. https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit

Sure, thank you

**Comment by leecalcote** [comment]
How did we do on the Websites call? All clear for merge?

**Comment by leecalcote** [comment]
I _think_ we're good to move forward here. Thanks much, @singh1203 👍

**Comment by singh1203** [comment]
> I _think_ we're good to move forward here. Thanks much, @singh1203 👍 

Thank you Sir @leecalcote

**Pull request merged**